MIGRATION.md

```diff
@@ -16,7 +16,8 @@
 - `CardComponent` configuration properies like `showsSecurityCodeField`, `showsStorePaymentMethodField`, `showsSecurityCodeField`, `stored`, and `excludedCardTypes` are moved to `CardComponent.Configuration` object.
 - Property `payment` moved from `PresentableComponent` to `PaymentComponent`.
 - `DropInActionComponent` renamed into `AdyenActionComponent` and moved to `AdyenActions` module. It could be used outside of `AdyenDropIn`.
-- Parameters `clientKey` and `environment` are required for `DropInComponent.PaymentMethodsConfiguration` constructor, and they are immutable.
+- `DropInComponent.PaymentMethodsConfiguration` renamed into `DropInComponent.Configuration.`
+- Parameters `clientKey` and `environment` are required for `DropInComponent.Configuration` constructor, and they are immutable.
```

Comment:
I think this line might not be relavant in v4 bc of `APIContext`

Reply:
good point..